.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  # Keep maven dependencies up-to-date
+  - package-ecosystem: maven
+    # Look for `pom.xml`  in the `root` directory
+    directory: "/"
+    # Check the maven registry for updates every day (weekdays)
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 0
+    target-branch: "master"    
```

Comment:
This does not affect `security` updates
- https://github.com/community/community/discussions/15027
- https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#target-branch

Reply:
@ekigamba I believe the `target-branch` supports version updates i.e. PRs to latest patches for dependencies. Security updates i.e. Vulnerabilities found similar to [this](https://github.com/opensrp/opensrp-server-core/security/dependabot) is only on the default branch. I am okay leaving the option for the sake getting version updates to multiple branches i.e. `v2` branch for example. Then the vulnerabilities can be extrapolated from the trivy scans